src/resources/js/components/ListPresentations.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <v-container>
+    <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+      <v-row align="center">
+      <v-card
+        class="mx-auto"
+        max-width="400"
+        tile
+      >
+        <v-list>
+          <v-subheader>Presentations</v-subheader>
+            <v-list-item
+              v-for="(presentation) in presentations.data"
+              :key="presentation.id"
+              inactive
+            >
+              <v-list-item-content>
+                <v-list-item-title v-html="presentation.title"></v-list-item-title>
+              </v-list-item-content>
+
+              <v-list-item-action>
+                <v-btn icon @click.prevent="goToPresentation(presentation.id)">
+                  <v-icon color="grey lighten-1">mdi-information</v-icon>
+                </v-btn>
+              </v-list-item-action>
+
+              <v-list-item-action>
+                <v-btn small color="error">
+                  quitter
+                </v-btn>
+              </v-list-item-action>
+
+              <v-list-item-action>
+                <v-btn small>
+                  action
+                </v-btn>
+              </v-list-item-action>
```

Comment:
unused buttons should not be here

Reply:
ok removed in [d2137](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/d2137dc3eae9168c2439f65a546d051e315424c3)